src/ConfirmedMap.js

```diff
@@ -20,11 +21,41 @@ class MbMap extends React.Component {
         };
     }
 
+    // Check if a date was more than two weeks ago
+    isOld(date) {
+        // Working with raw data, so try-catch just in case
+        try {
+            // 'DD/MM/YY' format
+            // Assume entries with incorrect formats are old
+            const eventDay = date.split("/");
+            if (eventDay.length !== 3 || eventDay === 'N/A') { return true; }
+
+            // Default constructor has current time
+            const today = new Date();
+            
+            // Day of the event. Transform to YYYY/MM/DD format
+            const day = eventDay[0].length == 1 ? '0' + eventDay[0] : eventDay[0];
```

Comment:
why do we need the 0? I think `new Date("2020-3-23")` also works

Reply:
TIL